.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "semantic/src/site"]
-	path = semantic/src/site
-	url = https://github.com/coding-blocks/semantic-site-config.git
+# [submodule "semantic/src/site"]
+# 	path = semantic/src/site
```

Comment:
blow away this submodule ! haha 
do not comment, delete the lines totally

Reply:
resolved via new commit